assets/src/application/ui/input/Number.tsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 import { InputNumber } from 'antd';
+import { InputNumberProps } from 'antd/lib/input-number';
 
-const NumberField = ({ input, ...rest }) => {
-	return <InputNumber {...input} {...rest} />;
```

Comment:
in this case, it didn't make sense to pass these separately

Reply:
It does make sense passing them separately. `input` object props have to be passed as props to `<InputNumber />`. Not just `input`, there is `meta` object as well but we haven't used it here.
For details about the shape of `input` and `meta`, see RFF [FieldRenderProps](https://final-form.org/docs/react-final-form/types/FieldRenderProps)